src/main/java/com/yahoo/sketches/SketchesException.java

```diff
@@ -0,0 +1,26 @@
+package com.yahoo.sketches;
```

Comment:
I actually like this suggestion.  However, to do this right would require creating several "SketchesException" sub-classes  and then redo all the thrown exceptions throughout the entire library.  I have implemented a simple model in master, but fully implementing this will take time.  I also don't feel we need to implement all the constructors when we only use one as this creates additional unused code that needs to be tested.  Rejected in this form. Implemented something simpler but better.